test/unit/helpers/dom/VerifyCustomFactorForm.js

```diff
@@ -0,0 +1,22 @@
+import Form from './Form';
+export default Form.extend({
+  backLink: function() {
+    return this.el('back-link');
+  },
+
+  buttonBar: function() {
+    return this.$('.o-form-button-bar');
+  },
+
+  hasErrorBox: function() {
+    return this.el('o-form-error-container').find('.infobox-error').length > 0;
+  },
+
+  hasSpinner() {
+    return this.$('.okta-waiting-spinner');
```

Comment:
this should return a boolean, so should be 
```
return this.$('.okta-waiting-spinner').length > 0;
```
otherwise we can rename the function to be `spinner` or `loadingSpinner` or something like that
(I vote for returning a boolean)